packages/accounts-password/password_server.js

```diff
@@ -67,6 +67,15 @@ Accounts._checkPassword = function (user, password) {
 
   if (! bcryptCompare(password, user.services.password.bcrypt)) {
     result.error = handleError("Incorrect password", false);
+  } else {
+    // password checks out, but user bcrypt may need update
+    if (user.services.password.bcrypt && Accounts._bcryptRounds > Number(user.services.password.bcrypt.substring(4, 6))) {
```

Comment:
Please wrap your code changes at 80 characters.

Reply:
No problem - done in b5cd99a